includes/features/drupal.org/addtocal/addtocal.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Add to Cal administration module
+  In order to ensure the site is functioning as expected
+  As administrator
+  I want to verify functionality of the Add to Cal module
+
+  Background:
+    Given I am logged in as a user with the "administrator" role
+
+  @safe @api
+  Scenario: Verify module is enabled
```

Comment:
I think we can remove this check in favor of having `given module is enabled` in later scenarios.